Gemfile

```diff
@@ -18,10 +19,12 @@ group :development do
   gem 'kramdown',      '~> 2.0'
   gem 'kramdown-man',  '~> 0.1'
 
-  gem 'redcarpet',       platform: :mri
-  gem 'yard',           '~> 0.9'
+  gem 'redcarpet', platform: :mri
```

Comment:
I'm actually a fan of vertical alignment. It makes it easier to edit the version requirements.